### ISSUE TITLE
Slack In: no chatId for event subscriptions

### ISSUE THREAD

**Issue by jacksbox** [guidone/node-red-contrib-chatbot#396]
When subscribing to slack events (like mentions etc.) 
The Slack receiver node fails with the error that the chatId can't be empty. 

The reason for this lays in the way how the channel is parsed for events. It expects an array of channels with the key `channel_ids` but most events only have a `channel` key with a single channelId.  

I opened a PR with a fix:
https://github.com/guidone/node-red-contrib-chatbot/pull/395

I skimmed the event payload list in the slack docs a bit and could not find events which use the `channels_id`, maybe this is not in use anymore?

**Comment by ff4500** [comment]
@jacksbox I'm assuming that this is related to an error that I'm getting when trying to read from a 
`reaction_added` or `reaction_removed` event as well. This is what the debug msg tells me:

```
Error: inbound message key "chatId" for transport slack is empty

See here: https://github.com/guidone/node-red-contrib-chatbot/wiki/Universal-Connector-node for an explanation about this error.
```

Running the newest version with your PR merged it, it looks like. Seems like the same issue, since no chatId is being set. This is from the basic Slack Receiver node piped straight to a debug node, nothing else.

**Comment by jacksbox** [comment]
@ff4500 yes, this looks like a similar issue. My PR fixed it just for the the `event_callback` type. 
When I find some time I'll have a more detailed lock at it.
